.gitignore

```diff
@@ -274,6 +274,10 @@ tags
 /test/parallel/taskPar/taskIntents/src/*.d
 /test/parallel/taskPar/taskIntents/src/*.o
 
+/test/parsing/king-11/aaa*
+/test/parsing/king-11/longName.good
+/test/parsing/king-11/longNameHelper*
+
```

Comment:
This is not crucial, but we are increasingly moving `.gitignore` files from the root directory to the subdirectory in which the files are being ignored.  I'm not sure how we got into this strange practice of putting all `.gitignore`s in the top-level directory, but it's problematic (e.g., makes it harder to move tests around).  Speaking of which...

Reply:
made a local `.gitingore` in test folder for the same